### ISSUE TITLE
Keybase.md

### ISSUE THREAD

**Issue by allyssonviegas** [keybase/proofs#185]
### Keybase proof

I hereby claim:

  * I am allyssonviegas on github.
  * I am allyssonviegas (https://keybase.io/allyssonviegas) on keybase.
  * I have a public key whose fingerprint is 92D6 EC68 66B0 790F 434A  21D7 BCF4 FCD0 B617 E1E4

To claim this, I am signing this object:

```json
{
  "body": {
    "key": {
      "eldest_kid": "0101a5a6b9e18796d2ce4b3ad4d849f2039bc0aacd13d268544df252bbfa1b3c2ad80a",
      "fingerprint": "92d6ec6866b0790f434a21d7bcf4fcd0b617e1e4",
      "host": "keybase.io",
      "key_id": "bcf4fcd0b617e1e4",
      "kid": "0101a5a6b9e18796d2ce4b3ad4d849f2039bc0aacd13d268544df252bbfa1b3c2ad80a",
      "uid": "9537a49f84138a9a58c5d9c1d267ee19",
      "username": "allyssonviegas"
    },
    "service": {
      "name": "github",
      "username": "allyssonviegas"
    },
    "type": "web_service_binding",
    "version": 1
  },
  "ctime": 1600426655,
  "expire_in": 157680000,
  "prev": "d5beaf0dc67e4aa337b3f60126b1dce0801df1a1d272830bf8b30cad89527023",
  "seqno": 24,
  "tag": "signature"
}
```

with the key [92D6 EC68 66B0 790F 434A  21D7 BCF4 FCD0 B617 E1E4](https://keybase.io/allyssonviegas), yielding the signature:

```
-----BEGIN PGP MESSAGE-----

owGtUmtQlFUYXlYaaQEhKcBM0S8aIG7f/bJQhCSTaBMZ6TTpLOd85+zuJ+u3sBcu
gSJqMqvMAEINlt00itEYwcTEsIkZRwwDTKWAEYQKuuGIMJGhQN8y+q+Z/nT+vHPe
93meeZ533qrARTqDT/S2tKDmTbdSfS79BnUmdPB4MQHtqIgwFhM5eKFgG8JOlylH
QYSRICmSAhzgoYQpUZB4RMuYhQxALBJZyUyTjARlEgAZUQyieZFjWWSmORpCM6Ag
I9MAiSQg4gizolqwI9ehqC5NVqIRj2Ve5HlIChJpZhkW0BQSoGxmzTIiIU8JmMKs
RrTanV6GZg4CJ05Q7FpP+5gW7P0L/n/27V6QkzhGABpPZClGBBLgRJlDkkxpVAFj
SvICndihgu1YQwObrcjptKv5CrYAJ7EjjtBm+YqMveu9j7EoLqsb/hfPVZTrHRRg
aLovYYKKirRdasx87HAqdpUwUhpSdileDYonSZbmeY6LI3BhruLAJsWL4AReJLUX
R+Q6cL4miTi4FQMziWQtAQsAwwiQMfMkRfOQQjImRZJCZgpoEQVaZEhoFiFDytpa
JI4WSJohvKnyVDthpFnNKLBook7FogKX24GJHQaPnvHV+Rh0K0JX+/bjhK6vb26o
Psq8Cx4c4EN67/XpDA8HP+gsiQyad65snE3/dfO1iwf8d67ePNXzQtH4Wdxehegz
gXlvetJXPfIN2JtMTAY9dsg68KxbbE4qaO1YfHpDyl8fBL61+MLt8tmh0HsVvxDp
XbPjbUKWY+aE+7WQ3ucbV9nDDEcs9aVPJJ9U98Q0vt7/uIu6dCt1qGPf/izl58Kv
cIajxPCicK1pwIfbC5frksa+yAozVMUw7/UN6te3vKJsadsYfmjONH7WGvFT9+e1
dvqHiKaQR+2fMksPZte9cWzOltk08P2XPUpK9icRrUd2rcycCyytrm0J9q8MKh2t
Zzf9ff5V3wY8V4HypyeYht6EumwYd9senZQGP2q5c93i7jxweX307HyCpYFkpZyY
pg49+0xf95PxNer7allNnnK4fWnU+Cnrt2Gh7YnVIcsDarZFD9+YZu2/f1dbunWK
HwrefSXZcX5sRVTf0eER02BAcwWfk34xKXp0aN/b8VvK0/2skWHLyjzd5wY8toCS
ifAra9mZzq6rPU3r6p3DS2JLhIGMlNMfzpVJ5Zl/Tk77VTZdXndvfvHd/ta8d1BD
RprnqawThYr+Y3++N3hXbIQx/uk/6hOOVwTvZ2riWXyycs0xNLlsqHj7DJza/fJN
T0nn1azxCzWBOwtSYz/rKj03ObKnceSlySh9yZrUO8bn7vqFR04cPtM2uNY4tmjj
jbqiUz+OJrYl5tjU6/8A
=UUOk
-----END PGP MESSAGE-----

```

And finally, I am proving ownership of the github account by posting this as a gist.

### My publicly-auditable identity:

https://keybase.io/allyssonviegas

### From the command line:

Consider the [keybase command line program](https://keybase.io/download).

```bash
# look me up
keybase id allyssonviegas
```

**Comment by allyssonviegas** [comment]
I've been trying to verify my account several times, via CLI and also WEB .. it has been failing multiple times.
I was able to proof other identities such as twitter and reddit. Initially I thought it could be something related to the 2FA with yubikeys but even without 2FA it still fail.

Please advise. 

Allysson Viegas

**Comment by chindraba-work** [comment]
To prove your identity on GitHub you have to do so by placing the proof in "your" account. Placing it here is like saying you own this account (github.com/keybase), which you don't. Your proof, once done, will be in https://gist.github.com/allyssonviegas somewhere.

**Comment by allyssonviegas** [comment]
Hello @chindraba-work 
Many thanks for the prompt response. I revoked the previous attempt and started again.. here is what I did.
1. created a repo called keybase
2. went to web and created a proof request.
3. placed the proof onto the keybase repo
https://github.com/allyssonviegas/keybase
4. I added the proof to Gist
https://gist.github.com/allyssonviegas

I still can't see my account verified.
Kindly let me know what I am missing here.

Cheers,
AV
